chatserver.py

```diff
@@ -217,9 +217,18 @@ def fileno(self):
     def send(self, data):
         self.socket.send(data)
 
+class IP_add:
+    def getIP(self):
+        s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
```

Comment:
Do you mean `s = socket.socket`? I think the indentation is okay. It runs without any error.